content/en/docs/instrumentation/cpp/manual.md

```diff
@@ -2,10 +2,24 @@
 title: Manual Instrumentation
 linkTitle: Manual
 weight: 30
+description: Manual instrumentation for OpenTelemetry C++
 ---
 
-Manual instrumentation is the process of adding observability code to your
-application.
+[Manual instrumentation](https://opentelemetry.io/docs/concepts/instrumentation/manual/)
+is the process of adding observability code to your application yourself. If you
+are developing a standalone process or service, then you would take a dependency
+on the API and the SDK.
```

Comment:
Start a new paragraph:

```suggestion
is the process of adding observability code to your application yourself.

If you
are developing a standalone process or service, then you would take a dependency
on the API and the SDK.
```

Also, I can't quite make sense of the wording of the last sentence.

Reply:
I reordered the original sentence, because most people come because the develop a standalone service, although that makes things harder to understand. Let me think about that for a little bit longer (or if anyone has good suggestions on the wording, please do :))

Reply:
I rewrote the whole text, outdating this discussion, will mark it as resolved